test/node/features/connect.feature

```diff
@@ -22,16 +22,18 @@ Scenario: a synchronous thrown error in a route
   And the event "request.httpMethod" equals "GET"
 
 Scenario: an asynchronous thrown error in a route
-  Then I open the URL "http://connect/async"
+  Then I open the URL "http://connect/async" tolerating any error
   And I wait to receive an error
   Then the error is valid for the error reporting API version "4" for the "Bugsnag Node" notifier
   And the event "unhandled" is true
   And the event "severity" equals "error"
-  And the event "severityReason.type" equals "unhandledErrorMiddleware"
+  And the event "severityReason.type" equals "unhandledException"
```

Comment:
because the error is captured by the top-level handler where we are now able to add request context if available